### ISSUE TITLE
xDrip unable to import blood tests values with "MEALTYPE:5" ="Bedtime"

### ISSUE THREAD

**Issue by diagnoe4** [NightscoutFoundation/xDrip#791]
I have a accu-check aviva connect. It allows the user to set one out of 6 mealtypes:
- Before meal
- After meal
- Fasting
- Bedtime
- Other
- No entry

Whenever the value "Bedtime" is selected, xDrip is unable to import the value and writes the following into the log:
BluetoothGlucoseMeter: Received context packet but we're not sure what its for: Context: Sequence: 2467 MEALTYPE: 5

I can confirm that all other "mealtypes" work.

**Comment by Navid200** [comment]
@diagnoe4 Is this still the case?
It's been two years.  Is there a need for an update to the description?
Thanks

**Comment by diagnoe4** [comment]
Hi, I just tried it again and still not able to import.

Xdrip version: 344702e-2021.02.11
Meter is a AccuChek Guide. Same error message (different sequence number).

Workaround: I'm not using that flag anymore.

**Comment by Navid200** [comment]
You import what to what?
This is a blood glucose meter, right?
So, I suppose you want to import a blood glucose meter reading?
You import the reading to what?  xDrip?  What for, calibration?  Is it Bluetooth?  If not, how is the data transferred?

**Comment by diagnoe4** [comment]
I have an accu-chek guide set up as a bluetooth blood glucose meter. The meter automatically sends readings to xdrip. On the meter, I can select those "meal types" I listed above. All but the "bedtime" are imported fine into xdrip. If I select "bedtime", they do not get imported and do not show up. The error message I listed is visible in the log.

I have xdrip set up to automatically calibrate with readings received through bluetooth.

All apart from the above works fine.
